cmd/process-agent/main_common.go

```diff
@@ -117,44 +52,54 @@ to your datadog.yaml file.
 Exiting.`
 )
 
-func runAgent(exit chan struct{}) {
+// main is the main application entry point
+func main() {
```

Comment:
Is this `main` function ever called? Isn't the main entry point the `rootCmdRun` command?

Reply:
`main` is still the entry point of inititialization. The order of execution:
`main` => `rootCmd` => `rootCmdRun` => `runAgent`.

Reply:
Oh, that makes sense. I was missing the `runcmd.Run(rootCmd)` inside the `os.Exit()` call. Thanks for clarifying it, Ivan!